wallet/chainntfns.go

```diff
@@ -817,6 +808,9 @@ func (w *Wallet) VoteOnOwnedTickets(ctx context.Context, winningTicketHashes []*
 	// be fixed somehow, but this should be stable for networks that
 	// are voting at normal block speeds.
 
+	defer w.lockedOutpointMu.Unlock()
+	w.lockedOutpointMu.Lock()
```

Comment:
we absolutely want to avoid needing this lock for creating votes. votes can never be double spent (except in a different fork, and dcrd has rules to explicitly allow these "double spends"), plus it will significantly hurt voting latency.